src/MusicalScore/Graphical/MusicSheetCalculator.ts

```diff
@@ -303,7 +303,7 @@ export abstract class MusicSheetCalculator {
     }
 
     protected layoutVoiceEntry(voiceEntry: VoiceEntry, graphicalNotes: GraphicalNote[],
-                               graphicalStaffEntry: GraphicalStaffEntry, hasPitchedNote: boolean): void {
+                               graphicalStaffEntry: GraphicalStaffEntry, hasPitchedNote: boolean, staffIndex: number): void {
```

Comment:
better not extend the function paramters but use: voiceEntry.ParentSourceStaffEntry.ParentStaff.idInMusicSheet to get the staffIndex.